DEVELOPER_GUIDE.md

```diff
@@ -162,3 +163,116 @@ $ yarn lint --fix
 ```
 
 `Husky` precommit hook is used to automatically run `yarn lint`, please fix the files according to lint result before commiting code changes (run `yarn lint --fix` for fixable errors, or manully fix code according to error messages). If you have any doubts on `ESLint` rules, feel free to [open an issue](issues).
+
+## Orchestrator Remote Test Workflow
+
+### Remote Cypress Test Runner - remoteCypress.sh
+
+`remoteCypress.sh` is a shell script that triggers GitHub workflow runners within a specified repository which runs Cypress integration tests on a remote OpenSearch/Dashboards cluster.
+
+#### Usage
+
+`./remoteCypress.sh -r REPO -w GITHUB_WORKFLOW_NAME -o OS_URL -d OSD_URL -b BRANCH_REF -i BUILD_ID`
+
+#### Arguments
+
+* -r REPO: Name of the repository in {owner}/{repository} format.
+* -w GITHUB_WORKFLOW_NAME: Name of the GitHub workflow file name with .yml extension that contain jobs that run Cypress tests in the component repository. For example, main.yaml.
```

Comment:
* -w GITHUB_WORKFLOW_NAME: Name of the GitHub workflow file with .yml extension that contains the jobs that run Cypress tests in the component repository. For example, main.yaml.